src/types/gecko-profile.js

```diff
@@ -191,6 +192,7 @@ export type GeckoProfileMeta = {|
 export type GeckoProfile = {|
   meta: GeckoProfileMeta,
   libs: Lib[],
+  pages: PageList,
```

Comment:
I think this needs to be `pages?` because this version of the profile format may not necessarily contain pages.